### DISCUSSION TITLE
OSHMI 6.26

### DISCUSSION THREAD

**Discussion by Krrorr** [riclolsen/OSHMI#8] [General]
Dear Riclolsen,
Thanks for sharing such a software in OS,
I read the docs/manuals multiple times, last I checked that I memorized 70% of them in last 20 days, but couldn't figure out things from them, I think they are general and I am lazy, would like to seek your help on the following:
1. Where can I add the status and command tags of DS & CB in existing svgs, I just want to view my signals on your created svgs. First practice on existing svgs, then I will create my own.
2. Does OSHMI has authentication levels like operator, engineer and admin, or something similar?, if so how can I activate it?
3. How to add details for tag's alarms and events & where?
4. Why I can't see your configuration on the objects in the existing svgs, how can I be able to see your configuration details of digital objects in existing svgs?, They work perfectly fine, but I couldn't find where is the configuration details, so I change the signals and some text, then it will work similar but for my signals.

Best Regards

**Comment by riclolsen** [comment]
Hi Krrorr,
1. Command tags are not in the SVG. Command tags are related to supervised tags using the SUPCMD column in the point_list.txt. When clicked the supervised object in the SVG the command will be available. You can link commands directly to SVG events using the directCommandExec function (page 22 of doc).
2. No. You can use Nginx auth to restrict access. RBAC is available on the JSON-SCADA project.
3. Messages for alarms are configured in the point_list.txt file.
4. Configuration of points are in the point_list.txt file, not in SVG.
Best regards,
Ricardo

**Reply by Krrorr** [reply]
Dear Riclolsen,
Thanks for the support,
1. OK I should add the supervised status tag to the CB / DS any digital object, the software will take its commands from point_list.txt.
2. OK, understood.
3. OK, I will check them and get back to you if I needed help.
4. I meant the interconnection between point list and SVG, anyway I got it.
Best regards,
Krrorr

**Comment by Krrorr** [comment]
Dear Riclolsen,

How to change the disconnector's (which has larger + sign) shape to change according to OFF / ON values of the signal, when the signal is OFF it should appear horizontal "---" and when ON it should appear "|", but it doesn't, now it is just a large + sign whatever the signal is. I am working on the coloring of them but don't know how that large plus sign to change with signal's value, please help.

In the existing SVGs that you have prepared, these things work fine, I supposed that the only job for me would be to change/replace the signals/tags in the SVGs with my new tags, and these SVGs would work for me, but now I see, I have to configure the coloring, shape changes too after replacing the existing objects' tags with my tags. That's something that confuses me every now and then.

Best regards,
Krrorr

**Reply by riclolsen** [reply]
A plus sign probably means the tag has no value acquired (not updated).
Check the protocol connection, logs, point_list and webserver window.

**Comment by Krrorr** [comment]
Dear Riclolson,
Wish you are doing great,

1. There are some single point commands which doesn't have related status points, the configuration manager xls file doesn't accept such commands, any instructions how can I overcome this issue of mine in OSHMI?
2. I used config_viewers.js file for styling but I couldn't reach even close to your already built examples in SVG files for example the Qualifier:failed style..., but I couldn't find from which file are those styling come from, can you help me that please.

Thanks

**Comment by Krrorr** [comment]
Dear Riclolson, 
Need your take on the followings:

1. How can I show the TCP IP connection of one OSHMI with another device in OSHMI svg?, by ping or so. It's like a network diagram, which shows if the devices are connected or disconnected.
2. Can we add a button or so to fit the complete drawing of svg to the width & height of actual screen?, I mean, with mouse we can zoom in & zoom out as we want, having a button which by clicking it, it adjust the width & height of substation's SLD to hardware monitor's width & height. Some calls it "Screen to Monitor Zoom" button. 

Best regards, 
Krrorr
